### ISSUE TITLE
Reading https://pypi.org/simple/psutil/

### ISSUE THREAD

**Issue by honphilemon** [xnl-h4ck3r/xnLinkFinder#3]
Just got stuck on this....

**Comment by xnl-h4ck3r** [comment]
Hi @honphilemon 
What do you mean exactly? You'll need to give some more detail to help you. Were you running `setup.py`, or `xnLinkFinder.py` or running `pip install psutil` to install it?
What OS are you on, what distro? What inout did you give and what was the output exactly?

**Comment by honphilemon** [comment]
> Hi @honphilemon What do you mean exactly? You'll need to give some more detail to help you. Were you running `setup.py`, or `xnLinkFinder.py` or running `pip install psutil` to install it? What OS are you on, what distro? What inout did you give and what was the output exactly?

**Comment by xnl-h4ck3r** [comment]
Yeah, it's more a tool for coverage than speed really. Unfirtunately archive.org isn't set up to go really fast on this without causing issues. I'm glad you think it's a great tool. Just let me know if you get any issues, or have any suggestions for improvement.
Many thanks
Xnl
